docs/en/reference/supporting-other-databases.rst

```diff
@@ -28,9 +28,9 @@ contributed back to Doctrine to make it an even better product.
 Implementation Steps in Detail
 ------------------------------
 
-1. Add your driver shortcut to class-name `Doctrine\DBAL\DriverManager`.
+1. Add your driver shortcut to class-name ``Doctrine\DBAL\DriverManager``.
 2. Make a copy of tests/dbproperties.xml.dev and adjust the values to your driver shortcut and testdatabase.
 3. Create three new classes implementing ``\Doctrine\DBAL\Driver\Connection``, ``\Doctrine\DBAL\Driver\Statement``
    and ``Doctrine\DBAL\Driver``. You can take a look at the ``Doctrine\DBAL\Driver\OCI8`` driver.
-4. You can run the testsuite of your new database driver by calling "phpunit -c .". You can set your own settings in the phpunit.xml file.
+4. You can run the test suite of your new database driver by calling ``phpunit -c .``. You can set your own settings in the phpunit.xml file.
```

Comment:
The `-c .` arguments look redundant. PHPUnit will try to locate the configuration file in the current directory by default.

Reply:
Yes, you don't need `-c` when you're starting phpunit from the same directory.

Reply:
Agree, removed.